Update user-profile-structure.md

rkhoriander opened the pull request:
Add data types to user profile info

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update user-object.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander commented:
https://docs-content-staging-pr-7220.herokuapp.com/docs/rules/references/user-object
https://docs-content-staging-pr-7220.herokuapp.com/docs/users/references/user-profile-structure

rkhoriander committed:
Update user-object.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update _user-prop-identities.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Create user-data-types.md

rkhoriander committed:
Rename user-data-types.md to _user-prof-attributes.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update and rename _user-prof-attributes.md to _user-prop-identities.md

erjohnson commented:
The 3-column table looks odd, wonder if there's a way we can make it look less scrunched

rkhoriander committed:
Update user-object.md

rkhoriander committed:
Update user-object.md

rkhoriander committed:
Update user-object.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander committed:
Update query-syntax.md

rkhoriander committed:
Update query-syntax.md

rkhoriander committed:
Update query-syntax.md

rkhoriander committed:
Update user-object.md

rkhoriander committed:
Update user-profile-structure.md

rkhoriander commented:
@erjohnson - I was following the legacy format on this page:
https://auth0.com/docs/users/search/v3/query-syntax

So I've added it to the PR and fixed the column widths across all of the pages. See what you think now.
